cloudmarker/events/azsqldatabasedisabledtdeevent.py

```diff
@@ -0,0 +1,95 @@
+"""Microsoft Azure SQL DB Transparent Data Encryption (TDE) event.
+
+This module defines the :class:`AzSQLDatabaseDisabledTDEEvent` class that
+identifies if a SQL Database has TDE disabled . This plugin works on the
+SQL DB properties found in the ``ext`` bucket of ``sql_db`` records.
+"""
+
+
+import logging
+
+from cloudmarker import util
+
+_log = logging.getLogger(__name__)
+
+
+class AzSQLDatabaseDisabledTDEEvent:
```

Comment:
I think a more handy name would be `AzSQLDatabaseTDEEvent`. Our plugin names have been devoid of the status so far ("disabled" or "enabled"), so we can continue that style by removing `Disabled` from the name. Note: An update to this would require an update in the module filename too.

Reply:
Renamed to `AzSQLDatabaseTDEEvent `